docs/MODULES.md

```diff
@@ -6,30 +6,38 @@ List of Module Documentations
 1.  [Crop](#crop-module)
 2.  [Segmented-Colormap](#segmented-colormap-module)
 3.  [FisheyeGl](#fisheyeGl-module)
-4.  [Average](#average-module)
-5.  [Blend](#blend-module)
-6.  [Blur](#blur-module)
-7.  [Brightness](#brightness-module)
-8.  [Channel](#channel-module)
-9.  [Colorbar](#colorbar-module)
-10. [Colormap](#colormap-module)
-11. [Contrast](#contrast-module)
-12. [Convolution](#convolutioon-module)
-13. [DecodeQr](#decodeQr-module)
-14. [Dynamic](#dynamic-module)
-15. [Edge-Detect](#edge-detect-module)
-16. [Gamma-Correction](#gamma-correction-module)
-17. [Gradient](#gradient-module)
-18. [Histogram](#histogram-module)
-19. [Import-image](#import-image-module)
-20. [Invert](#invert-module)
-21. [Ndvi](#ndvi-module)
-22. [Ndvi-Colormap](#ndvi-colormap-module)
-23. [Overlay](#overlay-module)
-24. [Resize](#resize-module)
-25. [Rotate](#rotate-module)
-26. [Saturation](#saturation-module)
-
+4.  [Add QR](#Add-QR-module)
+5.  [Average](#average-module)
+6.  [Blend](#blend-module)
+7.  [Blur](#blur-module)
+8.  [Brightness](#brightness-module)
+9.  [Channel](#channel-module)
+10. [Colorbar](#colorbar-module)
+11. [Colormap](#colormap-module)
+12. [Contrast](#contrast-module)
+13. [Convolution](#convolutioon-module)
```

Comment:
```suggestion
13. [Convolution](#convolution-module)
```
You can commit this suggestion directly from the browser. No need to make changes manually.